### DISCUSSION TITLE
Automatic table update SQLite and CSV export

### DISCUSSION THREAD

**Discussion by ruham20** [sqlitebrowser/sqlitebrowser#3134] [General]
Hi, maybe someone can help me I want to update (refresh) automatically the table and if it is possible to export CSV file at the same time?any help will be appreciated.
![2](https://user-images.githubusercontent.com/109856376/184678826-e88701e9-eddc-43f5-8fef-94b6d965b388.png)
![1](https://user-images.githubusercontent.com/109856376/184678833-da06897f-ab0f-4dbb-9b3d-5f5d79f44387.png)

**Comment by chrisjlocke** [comment]
Update the table from what?  Is something else editing the data (so the update has already taken place?) or do you mean you want to update the data from an external file?
DB4S has no means of automatically doing anything, but you can 'automate' using something like AutoIt (https://www.autoitscript.com/site/) which simulates keypresses and mouse-clicks, etc.
Alternatively, as SQLite is so open, you can write your own script in a myriad of programming languages.

**Reply by ruham20** [reply]
Thank you, I will try. I only want to refresh the data. Can you know how can I connect remotely ?

**Reply by chrisjlocke** [reply]
Refresh the data from what?  A csv file?
You can use many popular programming languages from C, C++, Python, Java, .NET to Go to connect to a SQLite database.  I can give you an example of how to connect using VB.Net as that's the only programming language I'm confident in.  You could easily translate that to C#.
